src/dbg/debugger_commands.cpp

```diff
@@ -142,6 +142,11 @@ CMDRESULT cbDebugRun(int argc, char* argv[])
     // Don't "run" twice if the program is already running
     if(dbgisrunning())
         return STATUS_ERROR;
+    if(!ThreaderIsAnyThreadActive())

```

Comment:
If you suspend all the threads and click "run", the debugger would enter a "running" state but no thread actually runs.